### PR TITLE
cis hardening addon

### PR DESCRIPTION
**Pull request by ktsakalozos**
This addon will apply the CIS hardening recommendations on the node it is called upon and will also install kube-bench as a plugin to be called via `sudo microk8s kube-bench`.

### CODE REVIEW

**Review by berkayoz — COMMENTED** [review]
Seems like we have an issue on strict testing. I also feel like the assertions in the cis tests feel a bit broad, something else could fail or numbers might change. Although for a starting PR, this looks okay overall.

**Review by berkayoz — COMMENTED** [review]
Looks good overall, but I'm seeing `73 checks PASS` `2 checks FAIL` for both the classic and strict tests. I believe there is an issue in our [workflow file](https://github.com/canonical/microk8s-core-addons/blob/f44892bb5fcae23c0a7329b96652c0015e181064/.github/workflows/run-tests.yml#L34-L36), we are not passing the `STRICT=yes` while calling pytest which in turn doesn't actually do strict specific testing. Which in turn shows classic tests succeeded although the expected output was `74 checks PASS`.

File: tests/validators.py
Comment (berkayoz) [inline tests/validators.py:1]: I think we need to set STRICT = yes. Not under time pressure. I don't see anywhere where this environment variable is set for core addons tests on workflow files.

Comment (ktsakalozos) [reply tests/validators.py:1]: I think it is correct now @berkayoz, let me know.

**Pull request merged**